_data/Prady029/Prady029.md

```diff
@@ -0,0 +1,7 @@
+---
+name: PRADYUMNA KUMAR SAHOO # No longer than 28 characters
+institution: Central UNiversity of Rajasthan 🚩 # no longer than 58 characters
```

Comment:
Change capital `N` to small `n`? UNiversity to University.

Reply:
Thank you for reviewing. I shall make amends. #Shoib you are doing a great job :)